apps/condo/domains/property/gql.js

```diff
@@ -5,13 +5,61 @@
  */
 
 const { generateGqlQueries } = require('@condo/domains/common/utils/codegeneration/generate.gql')
+const { ADDRESS_META_SUBFIELDS_QUERY_LIST } = require('@condo/domains/common/utils/addressApi/addressMetaGql')
 const { gql } = require('graphql-tag')
 
 const COMMON_FIELDS = 'id dv sender { dv fingerprint } v deletedAt organization { id name} newId createdBy { id name } updatedBy { id name } createdAt updatedAt'
-
-const PROPERTY_FIELDS = `{ name address addressMeta type ticketsInWork ticketsClosed unitsCount map ${COMMON_FIELDS} }`
+const PROPERTY_MAP_JSON_FIELDS = 'dv type sections { id type index name preview floors { id type index name units { id type name label preview } } }'
+const PROPERTY_FIELDS = `{ name address addressMeta { ${ADDRESS_META_SUBFIELDS_QUERY_LIST} } type ticketsInWork ticketsClosed unitsCount map { ${PROPERTY_MAP_JSON_FIELDS} } ${COMMON_FIELDS} }`
 const Property = generateGqlQueries('Property', PROPERTY_FIELDS)
 
+const PROPERTY_MAP_GRAPHQL_TYPES = `
+    scalar building
```

Comment:
we don't need scalar! All our API have only two scalars: JSON and Upload :)

Reply:
Show me, please, how can I ensure `type` fields for `sections`, `floors` and `units` to have constant values for following query:

```graphql
{
  allProperties {
    map {
      type
      sections {
        type
        floors {
          type
          units {
            type
          }
        }
      }
    }
  }
}
```

Now it correctly responds with following:

```json
{
  "data": {
    "allProperties": [
      {
        "map": {
          "type": "building",
          "sections": [
            {
              "type": "section",
              "floors": [
                {
                  "type": "floor",
                  "units": [
                    {
                      "type": "unit"
                    },
                  ]
                }
              ]
            }
          ]
        }
      }
    ]
  }
}
```